mopidy/http/handlers.py

```diff
@@ -108,28 +131,37 @@ class WebSocketHandler(tornado.websocket.WebSocketHandler):
     clients: ClassVar[set[WebSocketHandler]] = set()
 
     @classmethod
-    def broadcast(cls, msg, io_loop):
+    def broadcast(
+        cls,
+        msg: Union[bytes, str, dict[str, Any]],
+        io_loop: tornado.ioloop.IOLoop,
+    ) -> None:
         # This can be called from outside the Tornado ioloop, so we need to
         # safely cross the thread boundary by adding a callback to the loop.
         for client in cls.clients.copy():
             # One callback per client to keep time we hold up the loop short
             io_loop.add_callback(functools.partial(_send_broadcast, client, msg))
 
-    def initialize(self, core, allowed_origins, csrf_protection):
+    def initialize(
+        self,
+        core: CoreProxy,
+        allowed_origins: set[str],
+        csrf_protection: Optional[bool],
+    ) -> None:
         self.jsonrpc = make_jsonrpc_wrapper(core)
         self.allowed_origins = allowed_origins
         self.csrf_protection = csrf_protection
 
-    def open(self):
+    def open(self, *_args: str, **_kwargs: str) -> Optional[Awaitable[None]]:
```

Comment:
Can I double check this one with you?

Reply:
It is based on the signature in the superclass:

![screenshot_2023-08-20_18-09-24](https://github.com/mopidy/mopidy/assets/43726/58b021e9-b9a2-4ff0-b416-2d0d378d884f)